src/frontend/routes/Router.js

```diff
@@ -0,0 +1,14 @@
+import { Routes, Route } from "react-router-dom";
+import { Home, SigninPage, SignupPage } from "../pages";
+
+const Router = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Home />}></Route>
+      <Route exact path="/signin" element={<SigninPage />}></Route>
+      <Route exact path="/signup" element={<SignupPage />}></Route>
```

Comment:
Suggestion: remove any one route. It is repeating.

Reply:
Didn't get you which route you are saying as repeated